tests/e2e/browser/複数選択.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { test, expect, Page } from "@playwright/test";
+import { toggleSetting, navigateToMain } from "../navigators";
+
+test.beforeEach(async ({ page }) => {
+  const BASE_URL = "http://localhost:5173/#/home";
+  await page.setViewportSize({ width: 800, height: 600 });
+  await page.goto(BASE_URL);
+
+  await navigateToMain(page);
+  await page.waitForTimeout(100);
+  await toggleSetting(page, "複数選択");
+
+  await prepareAudioCells(page, 4);
+});
+
+const ctrlLike = process.platform === "darwin" ? "Meta" : "Control";
+/**
+ * アクティブなAudioCellと選択されているAudioCellを取得する。
+ * 戻り値のインデックスは1から始まる。（nth-childのインデックスと揃えるため）
+ */
+async function getSelectedStatus(page: Page): Promise<{
+  active: number;
+  selected: number[];
+}> {
+  const selectedAudioKeys = await page.evaluate(() => {
+    const audioCells = [...document.querySelectorAll(".audio-cell")];
+    let active: number | undefined;
+    const selected: number[] = [];
+    for (let i = 0; i < audioCells.length; i++) {
+      const audioCell = audioCells[i];
+      if (audioCell.classList.contains("active")) {
+        active = i + 1;
+      }
+      if (audioCell.classList.contains("selected")) {
+        selected.push(i + 1);
+      }
+    }
+
+    return { active, selected };
+  });
+  if (selectedAudioKeys.active === undefined) {
+    throw new Error("No active audio cell");
+  }
+  // return selectedAudioKeys; だと何故か型エラーを出すので、一旦オブジェクトを作って返す
+  // FIXME: なんとかする
+  return {
+    active: selectedAudioKeys.active,
+    selected: selectedAudioKeys.selected,
+  };
+}
+
+async function prepareAudioCells(page: Page, count: number) {
```

Comment:
すごく細かいですけど`add`の方が直感に合うかも。
```suggestion
async function addAudioCells(page: Page, count: number) {
```

Reply:
「count個のAudioCellを準備する」という意図で作りました（count=1だと追加されないのもそう）

Reply:
なるほどです、こっち見逃してました！
別のところでもコメント書きましたが、この意図だとAudioCellの数が1個の時以外の状態で実行すると予想外になってるかなと。